client/jetpack-connect/style.scss

```diff
@@ -1037,11 +1037,6 @@ $colophon-height: 50px;
 			color: var(--color-neutral-80);
 		}
 
-		.signup-form__submit {
-			border: 0;
-			box-shadow: none;
-		}
-
```

Comment:
These are all core buttons now so these styles would conflict with the core styles and are not needed.